pkg/providers/cloudstack/testdata/expected_results_main_no_machinetemplate_update_cp.yaml

```diff
@@ -389,18 +389,23 @@ metadata:
 spec:
   replicas: 3
   etcdadmConfigSpec:
-    etcdadmBuiltin: true
+    etcdadmBuiltin: false
     format: cloud-config
     cloudInitConfig:
       version: 3.4.16
       installDir: "/usr/bin"
+    etcdadmInstallCommands:
+      - echo this line exists so that etcdadmInstallCommands is not empty
+      - echo etcdadmInstallCommands can be removed once etcdadm bootstrap and controller fix the bug
```

Comment:
could you link a GH issue here?

Reply:
https://github.com/mrajashree/etcdadm-bootstrap-provider/issues/13